x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -62,57 +62,52 @@ export const getMetadataListRequestHandler = function (
   SecuritySolutionRequestHandlerContext
 > {
   return async (context, request, response) => {
-    try {
-      const agentService = endpointAppContext.service.getAgentService();
-      if (agentService === undefined) {
-        throw new Error('agentService not available');
-      }
+    const agentService = endpointAppContext.service.getAgentService();
+    if (agentService === undefined) {
+      throw new Error('agentService not available');
```

Comment:
Where does the error get handled? I see there wasn't a `catch` block before anyways, just curious